src/components/movieDetail/MovieDetail.js

```diff
@@ -1,32 +1,20 @@
 import React from 'react';
 import './MovieDetail.css';
-import movieData from '../../movieData';
 
 
-// const MovieDetail = () => {
-  
-//   console.log(movieData.movies[0])
-//   return (
-//     <section>
-//       <h2>Title:</h2>
-//     </section>
-//   )
-// }
-// {selectedDetails && <button onClick={clearSelection}>HOME</button>}
-
 const MovieDetail = ({movieInfo}) => {
+  console.log(movieInfo)
   return (
-    <section>
-      
+    <section> 
       <img className="backdrop" src={movieInfo.backdrop_path}/>
-      <h2>Title: {movieInfo.title}</h2>
-      <h3>Tagline: {movieInfo.tagline}</h3>
-      <p>Overview: {movieInfo.overview}</p>
+      <h2>{movieInfo.title}</h2>
+      <h3>{movieInfo.tagline}</h3>
+      <p>{movieInfo.overview}</p>
       <p>Genre: {movieInfo.genres}</p>
-      <p>Average Rating: {movieInfo.average_rating}</p>
-      <p>Budget: {movieInfo.budget}</p>
-      <p>Revenue: {movieInfo.revenue}</p>
-      <p>Runtime: {movieInfo.runtime}</p>
+      <p>Average Rating: {movieInfo.average_rating.toFixed(2)}</p>
+      <p>Budget: ${movieInfo.budget}</p>
+      <p>Revenue: ${movieInfo.revenue}</p>
+      <p>Runtime: {movieInfo.runtime} min</p>
```

Comment:
Good opportunity to practice destructuring here - it would save you from typing `movieInfo` as many times as you are 😄